test/e2e/specs/fixtures/block-transforms.json

```diff
@@ -0,0 +1,105 @@
+{
+	"core__archives":[],
+	"core__archives__showPostCounts":[],
+	"core__audio":[
+		"File"
```

Comment:
Any way to have this both be block names (plus attributes) or both be file names?